cpa/parsing.go

```diff
@@ -131,10 +131,14 @@ func parseBundle(bundle map[string]string, rules ...LintRule) (*Policy, error) {
 		return nil, fmt.Errorf("failed policy linting: %w", multiErr)
 	}
 
-	if shouldImportConfigHelpers(moduleMap) {
-		if err := helpers.AppendCircleCIConfigHelpers(moduleMap); err != nil {
-			return nil, fmt.Errorf("failed to import helper functions")
-		}
+	if hasImport(moduleMap, "data.circleci.config") {
```

Comment:
Can we pass this static value as a constant?
```suggestion
	if hasImport(moduleMap, CIRCLECI_CONFIG) {
```